apps/trackers/tests/test_integration.py

```diff
@@ -7,7 +7,7 @@
 from rest_framework import status
 
 from apps.trackers.jira.query import JiraPriority
-from apps.trackers.save import TrackerSaver
+# from apps.trackers.save import TrackerSaver
```

Comment:
I left it there so the reviewer can follow the implementation process or at least I though it will be better to understand :smile: no I see it is maybe rather confusing. It is already on **master** so I will just keep it the same way here